### DISCUSSION TITLE
How to deploy addy.io with docker?

### DISCUSSION THREAD

**Discussion by eleaner** [anonaddy/docker#277] [Q&A]
Hi,

Where I can find more detailed information on how to deploy addy with docker?
Do I set the same DNS as the non-docker version?
it is kind of hard for less techy person.

thanks

**Comment by anym001** [comment]
I would also like to use the Docker version and still have an unanswered question. 
Is it mandatory to have your own domain for the self-hosted version or is it still possible via anonaddy.com? (ANONADDY_DOMAIN)

**Reply by willbrowningme** [reply]
You must have you own domain to self-host.

**Reply by anym001** [reply]
Thank you for the clarification.

**Comment by eleaner** [comment]
I have own domain but still I am not sure how to properly set addy.io.
The only documentation I can find does not talk about docker
